install.sh

```diff
@@ -2173,24 +1638,61 @@ MARKETPLACE_DRIVER_S3_SCRIPTS="src/market_mad/remotes/s3/import \
             src/market_mad/remotes/s3/monitor \
             src/market_mad/remotes/s3/S3.rb"
 
-MARKETPLACE_DRIVER_COMMON_SCRIPTS="src/market_mad/remotes/common/lxd.rb \
-                                   src/market_mad/remotes/common/docker.rb"
-
 MARKETPLACE_DRIVER_LXC_SCRIPTS="src/market_mad/remotes/linuxcontainers/import \
             src/market_mad/remotes/linuxcontainers/delete \
-            src/market_mad/remotes/linuxcontainers/monitor"
-
-MARKETPLACE_DRIVER_TK_SCRIPTS="src/market_mad/remotes/turnkeylinux/import \
-            src/market_mad/remotes/turnkeylinux/delete \
-            src/market_mad/remotes/turnkeylinux/monitor"
-
-MARKETPLACE_DRIVER_DH_SCRIPTS="src/market_mad/remotes/dockerhub/import \
-            src/market_mad/remotes/dockerhub/delete \
-            src/market_mad/remotes/dockerhub/monitor"
-
-MARKETPLACE_DRIVER_REGISTRY_SCRIPTS="src/market_mad/remotes/docker_registry/import \
-                                     src/market_mad/remotes/docker_registry/delete \
-                                     src/market_mad/remotes/docker_registry/monitor"
+            src/market_mad/remotes/linuxcontainers/monitor \
+            src/market_mad/remotes/linuxcontainers/lxd.rb"
+
+#-------------------------------------------------------------------------------
+# Scheduler drivers, to be installed under $REMOTES_LOCATION/sched
+#   - Rank scheduler $REMOTES_LOCATION/scheduler/rank
+#   - OpenNebula DRS, $REMOTES_LOCATION/scheduler/one-drs
+#-------------------------------------------------------------------------------
+
+SCHEDULER_DRIVER_RANK_SCRIPTS="src/schedm_mad/remotes/rank/src/sched/place \
+            src/schedm_mad/remotes/rank/optimize"
+
+SCHEDULER_DRIVER_ONEDRS_SCRIPTS="src/schedm_mad/remotes/one_drs/place \
+            src/schedm_mad/remotes/one_drs/optimize"
+
+SCHEDULER_DRIVER_DUMMY_SCRIPTS="src/schedm_mad/remotes/dummy/place \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

SCHEDULER_DRIVER_DUMMY_SCRIPTS appears unused. Verify use (or export if used externally).